.docker/ubuntu-22.04/Dockerfile

```diff
@@ -16,11 +16,8 @@ RUN apt-get update && apt-get install --no-install-recommends -y apt-transport-h
     apt-add-repository ppa:lttng/stable-2.13 && \
     apt-get update && apt-get install -y tzdata && apt-get install -y \
     build-essential \
-    gcc-arm-linux-gnueabihf \
```

Comment:
why removing these? they are needed for cross compile

Reply:
gcc-multilib automatically removes these libraries, but leave these there makes installation failing

Reply:
then you can't have gcc-multilib in this docker file or you will need to figure out a way to make it coexist with these libs.

Reply:
found the way to cover in CMakeLists.txt without installing gcc-multilib